### DISCUSSION TITLE
iOS TestFlight subscriptions…

### DISCUSSION THREAD

**Discussion by Maddest-Max** [osmandapp/OsmAnd#16671] [General]
Is there a reason I have to resubscribe daily in the app for testing? It's not something I have to do in other apps I'm testing that require a subscription service for certain functionality. If it's supposed to keep people from just running the testing version all the time, that's what tester number limits are for. It's hugely inconvenient to have to do it everyday. If it needs to be limited, at least make it a week between.

**Comment by kashortiexda** [comment]
Did you get any answer on this.
I have OSMAnd Pro subscription and same here, have to re-subscribe daily, it’s ridiculously inconvenient especially when out of wifi/data/offline… which, given it’s predominantly an outdoors offline-focused app, that’s where it needs to be tested, makes it almost impossible to actually use 
I wish they could make the re-subscribe weekly or every two weeks (same concept as a free trial works)… I mean, they already got my yearly subscription money, so why make me “re-subscribe” daily ????

**Reply by Maddest-Max** [reply]
I never did. I figured it had to do with api calls for the weather or something. And as just a tester, I didn't push. If you're a paid subscriber though, someone should definitely take a look. I thought it was just a testing issue. Best of luck!

**Reply by dmpr0** [reply]
Hi, @kashortiexda!
If you have OsmAnd Pro, you just need to login into OsmAnd Cloud.
